xmtp_mls/src/groups/mod.rs

```diff
@@ -461,10 +467,12 @@ where
         // If not an existing group, the creator is a super admin and should publish the commit log
         // Otherwise, for existing groups, we'll never publish the commit log until we receive a welcome message
         let should_publish_commit_log = existing_group_id.is_none();
+
         let stored_group = StoredGroup::builder()
             .id(group_id.clone())
             .created_at_ns(now_ns())
             .membership_state(membership_state)
+            .conversation_type(conversation_type)
             .added_by_inbox_id(context.inbox_id().to_string())
```

Comment:
Previously sync groups didn't set this column, now they do. I don't think this will make a difference, so didn't try to add any special-case logic here